packages/dara-components/dara/components/common/dropzone.py

```diff
@@ -67,6 +69,7 @@ def __init__(
             - if a target is specified, can be used to customise how the `bytes` received are turned into a `DataFrame`
             - if a target is not specified, can be treated as a side effect function to run on the `bytes` received (to i.e. store on disk)
         :param on_drop: optional action triggered when a file is successfully uploaded
+        :param enable_paste: optional flag to enable pasting of text as files
```

Comment:
Could we make this a bit clearer? What it does is that it stops us adding the paste event listener. Which means mostly that the user can't cmd/ctrl + V or paste via paste option on the mouse context menu. I think a bit more just saying what it enables and what it can be useful for/when to avoid briefly would be useful.

Reply:
I realize I cut a bit too much context when translating from JS side to Python. Expanded on the description a bit more. Would this be helpful?

Reply:
Yes, looks good :)